app/controllers/space_contacts_controller.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class SpaceContactsController < AuthenticateController
+  before_action :set_space_contact, except: [:create]
```

Comment:
Another nice rails-thing that my controllers could definitely have used! Please let me know whenever you see that my code could be better and more rails-y when reviewing it - I want the code that goes into the project to work similiarliy.

Reply:
Will do :D